### PR TITLE
Impl hi-fi About/Video section

### LINKED ISSUE #41: Implement hi-fi About section for cmd-f
- Implement hi-fi Hero section for cmd-f according to the [design](https://www.figma.com/file/kWqvtaGbRQcFh6UvPE8ILt/cmd-f-2021?node-id=119%3A2)

### PR DESCRIPTION
**Pull request by yehee**
Fixes #41 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Works on staging site

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by kozr** [comment]
image
The hi-fi design looks different

File: src/components/video/Video.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                <Iframe
+                src="https://www.youtube.com/embed/HwSv6NsqDWM"
+                frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                allowfullscreen />
+                <div className="info">
+                    <div>
+                        <img src={logo} />
+                        <div>
+                            <p className="title">cmd-f 2020 | British Columbia's largest all-women* hackathon 🌱</p>
```
Comment (kozr) [inline src/components/video/Video.jsx:55]: For next year's sake, I would recommend having these text pulled from the Intro section on CMS.

Comment (kozr) [reply src/components/video/Video.jsx:55]: <p className="title">cmd-f 2020 | British Columbia's largest all-women* hackathon 🌱</p> and <p className="subtitle">nwPlus • Mar 21, 2020</p>

File: src/lib/Containers.jsx [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
   margin: ${p => p.margin};
 `
+export const SectionContainerWithBackground = styled(SectionContainer)`
+  background-color: #F9F6EF;
+  background-image: url(${p => p.src});
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  height: ${p => p.height};
+`
```
Comment (kozr) [inline src/lib/Containers.jsx:16]: I like this.

File: src/pages/index.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ import fireDb from '@utilities/firebase'
 import { serialize } from '@utilities/format'
 import FAQExpandable from '@components/faqTemplates/ExpandableWithCategories'
 import About from '@components/about/TwoColumnsAbout'
+import Video from '@components/video/Video'
 
 export default function Index({ flags: { faqFlag }, faq, faqConfig, example, about }) {
```
Comment (kozr) [inline src/pages/index.js:10]: Do you think we can refactor this object into a single useState hook?

**Review by kozr — DISMISSED** [review]

**Comment by kozr** [comment]
It looks wonderful, the video component is nicely made.

File: src/pages/index.js (hunk 3 above)
Comment (yehee) [reply src/pages/index.js:10]: Can you elaborate on this idea?

**Comment by yehee** [comment]
@kozr for some reason I'm having trouble exporting all the components related to this section's background, especially with the books component. Also getting some weird looking plants on the extracted background like such:
Screen Shot 2021-01-28 at 10 45 36 PM
which should actually look like this:
Screen Shot 2021-01-28 at 10 45 47 PM
Perhaps I'm not properly exporting the SVGs, how are people usually exporting components for background?

**Comment by kozr** [comment]
@yehee I usually direct export issues to design members. I always try and fail at fixing figma things :(

**Comment by yehee** [comment]
@kozr I reached out to Marisa and she was able to fix it for us!

**Review by kozr — COMMENTED** [review]
@yehee There is a merge conflict

**Comment by yehee** [comment]
@kozr Thanks for taking care of that! Should be ready for another run 😞  The mobile implementations are to come shortly after.

**Pull request merged**